locales-src/newtab.ftl

```diff
@@ -52,3 +52,78 @@ newtab-confirm-delete-history-p1 = Are you sure you want to delete every instanc
+newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+
+# Note: Bookmark is a noun in this case, "Remove bookmark".
+newtab-menu-remove-bookmark = Remove Bookmark
+# Note: Bookmark is a verb here.
+newtab-menu-bookmark = Bookmark
+
+## Context Menu - Downloaded Menu: In (newtab-menu-copy-download-link, newtab-menu-go-to-download-page), 
+## "Download" in these cases is not a verb, it is a noun. As in, "Copy the link that belongs to this downloaded item".
+newtab-menu-copy-download-link = Copy Download Link
+newtab-menu-go-to-download-page = Go to Download Page
+newtab-menu-remove-download = Remove from History
+
+## Context Menu - Download Menu: These are platform specific strings found in the context menu of an item that has
+## been downloaded. The intention behind "this action" is that it will show where the downloaded file exists on the file
+## system for each operating system.
+newtab-menu-show-file =
+    { PLATFORM() ->
+        [macos] Show in Finder
+        [windows] Open Containing Folder
+        [linux] Open Containing Folder
+       *[other] Show File
+    }
+newtab-menu-open-file = Open File
+
+## Card Tooltip: Action tooltip to open a context menu
+newtab-card-tooltip =
+    .title = Open menu
+
+## Card Labels: These labels are associated to pages to give
+## context on how the element is related to the user, e.g. type indicates that
+## the page is bookmarked, or is currently open on another device
+newtab-label-visited = Visited
+newtab-label-bookmarked = Bookmarked
+newtab-label-recommended = Trending
+newtab-label-saved = Saved to { -pocket-brand-name }
+newtab-label-download = Downloaded
+
+## Section Menu: These strings are displayed in the section context menu and are 
+## meant as a call to action for the given section.
+newtab-section-menu-remove-section = Remove Section
+newtab-section-menu-collapse-section = Collapse Section
+newtab-section-menu-expand-section = Expand Section
+newtab-section-menu-manage-section = Manage Section
+newtab-section-menu-manage-webext = Manage Extension
+newtab-section-menu-add-topsite = Add Top Site
+newtab-section-menu-add-search-engine = Add Search Engine
+newtab-section-menu-move-up = Move Up
+newtab-section-menu-move-down = Move Down
+newtab-section-menu-privacy-notice = Privacy Notice
+
+## Empty Section States: These show when there are no more items in a section. Ex. When there are no more Pocket story reccomendations, in the space where there would have been stories, this is shown instead.
+newtab-empty-section-highlights = Start browsing, and we’ll show some of the great articles, videos, and other pages you’ve recently visited or bookmarked here.
+# Variables:
+#  $provider (String): Name of the content provider for this section, e.g "Pocket".
+newtab-empty-section-topstories = You’ve caught up. Check back later for more top stories from { $provider }. Can’t wait? Select a popular topic to find more great stories from around the web.
```

Comment:
nit: " Ex. When there are no more Pocket story reccomendations, in the space where there would have been stories, this is shown instead." seems more apt above newtab-empty-section-topstories